app/routes/articles/ArticleListRoute.ts

```diff
@@ -34,8 +34,10 @@ const mapStateToProps = (state, props) => {
     }
   ).map((article) => ({
     ...article,
-    author: selectUserById(state, {
-      userId: article.author,
+    authors: article.authors.map((e) => {
+      return selectUserById(state, {
+        userId: e,
+      });
```

Comment:
You should _ideally_ use a selector for multiple users here. Not critical though